src/components/targeting/__tests__/TargetingContextProvider.test.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+import * as React from 'react';
+import { act } from 'react-dom/test-utils';
+
+import TargetingContextProvider from '../TargetingContextProvider';
+import { TargetingContext } from '../../../features/targeting';
+
+function Consumer() {
+    const { getTargetingApi } = React.useContext(TargetingContext);
+    const { shouldShow, toClose, onSeen } = getTargetingApi('msg');
```

Comment:
sorry can we make this `onClose` (i'm so sorry that my comment previously was inaccurate).